### PR TITLE
feat(issue-details): Adds in resources for ChunkLoadError

### PR DESCRIPTION
**Pull request by roggenkemper**
this pr adds in resources for `ChunkLoadErrors`. Is built so that future errors can be easily added as well. Since all of these will be "errors" as the IssueType, need to look at the title to figure out what kind of error it is. Do not merge until we have a real link in for the resources. 

Screenshot 2023-11-29 at 9 52 40 AM

### CODE REVIEW

File: static/app/utils/issueTypeConfig/errorConfig.tsx [hunk 1]
```diff
@@ -18,4 +20,19 @@ const errorConfig: IssueCategoryConfigMapping = {
   },
 };
 
-export default errorConfig;
+export const titleResourceMap = {
```
Comment (scttcper) [inline static/app/utils/issueTypeConfig/errorConfig.tsx:23]: a type here would be great

File: static/app/utils/issueTypeConfig/index.tsx [hunk 2]
```diff
@@ -47,6 +48,16 @@ const issueTypeConfig: Config = {
...
 };
 
+function getErrorResourceConfig(title: string) {
+  let errorTitle = '';
+
+  if (title.includes('ChunkLoadError')) {
+    errorTitle = ErrorTitles.CHUNK_LOAD_ERROR;
+  }
+  const resource = titleResourceMap[errorTitle];
+  return resource ?? {};
```
[the remark this answers is not in the document]
Comment (scttcper) [reply static/app/utils/issueTypeConfig/index.tsx:58]: nevermind object good

**Review by scttcper — APPROVED** [review]

File: static/app/utils/issueTypeConfig/errorConfig.tsx [hunk 3]
```diff
@@ -18,4 +20,19 @@ const errorConfig: IssueCategoryConfigMapping = {
...
+  [ErrorTitles.CHUNK_LOAD_ERROR]: {
+    resources: {
+      description: t(
+        'ChunkLoadErrors occur when the JavaScript chunks (bundles) that an application is trying to load encounter issues during the loading process. Some common causes are dynamic imports, version mismatching, and code splitting issues. To learn more about how to fix ChunkLoadErrors, check out these resources:'
+      ),
+      links: [
+        {
+          text: t('How to fix ChunkLoadErrors'),
+          link: 'https://sentry.io',
```
Comment (scttcper) [inline static/app/utils/issueTypeConfig/errorConfig.tsx:32]: either lets flag it or wait for the link

Comment (scefali) [reply static/app/utils/issueTypeConfig/errorConfig.tsx]: @scttcper it's ready now

**Pull request merged**